install/swiftly-install.sh

```diff
@@ -566,19 +585,27 @@ if [[ -f "$HOME_DIR/config.json" ]]; then
     detected_existing_installation="true"
     if [[ "$overwrite_existing_intallation" == "true" ]]; then
         echo "Overwriting existing swiftly installation at $(replace_home_path $HOME_DIR)"
```

Comment:
$HOME_DIR still seems to be pointing to $HOME/.local/share/swiftly and not $HOME/Library/Application Support/swiftly. See line 533

Reply:
Thanks, I'll update those locations too. This is all abstracted nicely in the Swift code through the Platform protocol, so hopefully we can start using that soon instead of this duplication.